### PR TITLE
TCS edits in Creating CL VM on Google Cloud Platform doc

### PR DESCRIPTION
**Pull request by MCamp859**
TCS edits to GCE guide for formatting, branding, and grammar.
Follow on to https://github.com/clearlinux/clear-linux-documentation/pull/505
@ttzeng  Please review to be sure technical details are still OK.

### CODE REVIEW

File: source/clear-linux/get-started/virtual-machine-install/gce.rst [hunk 1]
```diff
@@ -1,31 +1,46 @@
...
+instance of |CL-ATTR| on `Google Cloud Platform`_ (:abbr:`GCP (Google Cloud Platform)`).
 
-#. Sign in your Google account on the
+.. contents:: :local:
+   :depth: 1
+
+Prerequisites
+*************
+
+** TBD **
```
Comment (MCamp859) [inline source/clear-linux/get-started/virtual-machine-install/gce.rst]: @ttzeng  Can you provide details here?

File: source/clear-linux/get-started/virtual-machine-install/gce.rst [hunk 2]
```diff
@@ -135,67 +172,88 @@ instance of |CL-ATTR| on `Google Cloud Platform`_ (GCP).
...
    previous step, and the username associated with the SSH public key:
 
    .. figure:: figures/gce/42-ssh-vm.png
       :scale: 50 %
-      :alt: SSH log in to the Clear Linux VM
+      :alt: SSH login to the Clear Linux VM
+
+      Figure 20: SSH login to Clear Linux VM
+
+Related topics
```
Comment (MCamp859) [inline source/clear-linux/get-started/virtual-machine-install/gce.rst]: @ttzeng  Any other related topics that I should link here?

**Comment by MCamp859** [comment]
When merged, this PR closes issue https://github.com/clearlinux/clear-linux-documentation/issues/524.

File: source/clear-linux/get-started/virtual-machine-install/gce.rst (hunk 1 above)
Comment (ttzeng) [reply source/clear-linux/get-started/virtual-machine-install/gce.rst]: @MCamp859, the only prerequisites I can think of are (1) the reader should already have a Google account, and a billing account set up in GCP in Step 1; and (2) the user SSH key is generated and installed in the Linux PCs that is used to connect to the VMs in GCP (last bullet in Step 13, and Step 15)

File: source/clear-linux/get-started/virtual-machine-install/gce.rst (hunk 2 above)
Comment (ttzeng) [reply source/clear-linux/get-started/virtual-machine-install/gce.rst]: @MCamp859, maybe we can put **similar** topics here, though they are not related to Google cloud... they are tutorials running CL on [Microsoft Azure](https://clearlinux.org/documentation/clear-linux/tutorials/azure) and [Amazon AWS](https://clearlinux.org/documentation/clear-linux/tutorials/aws-web) clouds respectively.

**Comment by ttzeng** [comment]
> @MCamp859, changes you made look good. @ttzeng we need to close on the prerequisites and related topics sections. It's ok if there isn't anything that needs to be there. We can simply remove those sections.

Thanks @MCamp859 for the excellence works! I have added the demanding sections as my little 2 cents comment. (y)

**Comment by MCamp859** [comment]
Thanks @ttzeng for the feedback, I added your text.
@intelkevinputnam please take one more look then OK to merge.

**Comment by krisdale** [comment]
closes #524

**Pull request merged**